File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1165.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8')
var lines = input.split('\n')

n = parseInt(lines.shift())
for(j=0;j<n;j++){
    v = parseInt(lines.shift())
    div=0
    if(v==2){
        console.log(v+" eh primo")
    } else if(v==1){
        console.log(v+" nao eh primo")
    } else if(v%2==0){
        console.log(v+" nao eh primo")
    } else{
        for(i=3;i<v;i+=2){
            if(v%i==0){
                div=1
                break
            }
        }
        if(div==0){
            console.log(v+" eh primo")
        } else{
            console.log(v+" nao eh primo")
        }
    }
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1165.py
n = int(input())
for j in range(0, n):
    v = int(input())
    div=0
    if(v==2):
        print(str(v)+" eh primo")
    elif(v==1):
        print(str(v)+" nao eh primo")
    elif(v%2==0):
        print(str(v)+" nao eh primo")
    else:
        for i in range(3, v, 2):
            if(v%i==0):
                div=1
                break
        
        if(div==0):
            print(str(v)+" eh primo")
        else:
            print(str(v)+" nao eh primo")
        
    
